frontend/src/hooks/usePersistentTableState.ts

```diff
@@ -51,6 +51,19 @@ export const usePersistentTableState = <T extends QueryParamConfigMap>(
         return reorderObject(tableState, [...searchParams.keys()]);
     }, [searchParams, tableState, reorderObject]);
 
+    useEffect(() => {
```

Comment:
centralized limit setting to predefined values. even when no pagination bar is included this will work